frontend/app/src/pages/onboarding/create-tenant/index.tsx

```diff
@@ -132,16 +134,16 @@ export default function CreateTenant() {
 
       setTimeout(() => {
         if (result.type === 'cloud') {
-          // For cloud tenants, redirect back to organization page
-          window.location.href = `/organizations/${selectedOrganizationId}`;
+          const tenant = result.data as OrganizationTenant;
+          navigate(`/tenants/${tenant.id}/onboarding/get-started`);
+          return;
+        }
+
+        const tenant = result.data as Tenant;
+        if (tenant.version === TenantVersion.V1) {
+          navigate(`/tenants/${tenant.metadata.id}/onboarding/get-started`);
         } else {
-          // For regular tenants, use the existing onboarding flow
-          const tenant = result.data as any;
-          if (tenant.version === TenantVersion.V1) {
-            window.location.href = `/tenants/${tenant.metadata.id}/onboarding/get-started`;
-          } else {
-            window.location.href = `/onboarding/get-started?tenant=${tenant.metadata.id}`;
-          }
+          navigate(`/onboarding/get-started?tenant=${tenant.metadata.id}`);
```

Comment:
there was a reason for the hard-redirects here, but I'm struggling to remember what it was. it might be good to use `window.location.href = ` for this one just to be safe. we can change it when we remove the old v0 code